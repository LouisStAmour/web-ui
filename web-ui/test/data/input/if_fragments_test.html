<!DOCTYPE html>
<!--
Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
for details. All rights reserved. Use of this source code is governed by a
BSD-style license that can be found in the LICENSE file.
-->
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <script src="packages/web_ui/testing/testing.js"></script>
</head>
<body>
  <!-- Tests that we can have document fragmwents inside iteration. -->
  expect t - t:
  <template if="cond">
    t {{value}} t
  </template>
  <script type="application/dart">
  import 'dart:html';
  bool cond = true;
  var value = '-';
  main() {
    window.postMessage('done', '*');
  }
  </script>
</body>
</html>
