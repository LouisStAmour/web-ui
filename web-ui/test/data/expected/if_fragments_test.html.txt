Content-Type: text/plain
layer at (0,0) size 800x600
  RenderView at (0,0) size 800x600
layer at (0,0) size 800x34
  RenderBlock {HTML} at (0,0) size 800x34
    RenderBody {BODY} at (8,8) size 784x18
      RenderText {#text} at (0,0) size 140x18
        text run at (0,0) width 140: "expect t - t: "
      RenderText {#text} at (140,0) size 20x18
        text run at (140,0) width 20: "t "
      RenderText {#text} at (160,0) size 10x18
        text run at (160,0) width 10: "-"
      RenderText {#text} at (170,0) size 20x18
        text run at (170,0) width 20: " t"
      RenderText {#text} at (0,0) size 0x0
      RenderText {#text} at (0,0) size 0x0
